fip.c

```diff
@@ -130,6 +145,27 @@ static uuid_t const uuid_list[] = {
 		0xd6, 0xd0, 0xee, 0xa7, 0xfc, 0xea, 0xd5, 0x4b,
 		0x97, 0x82, 0x99, 0x34, 0xf2, 0x34, 0xb6, 0xe4
 	},
+	[FBI_BL2_DATA] = {
+		0xf4, 0x1d, 0x14, 0x86, 0xcb, 0x95, 0xe6, 0x11,
+                0x84, 0x88, 0x84, 0x2b, 0x2b, 0x01, 0xca, 0x38
+	},
+	[FBI_BL30_DATA] = {
+		0x48, 0x56, 0xcc, 0xc2, 0xcc, 0x85, 0xe6, 0x11,
+                0xa5, 0x36, 0x3c, 0x97, 0x0e, 0x97, 0xa0, 0xee
+	},
+	[FBI_BL31_DATA] = {
+		0xca, 0xaf, 0xb0, 0x33, 0xce, 0x85, 0xe6, 0x11,
+                0x8c, 0x32, 0x00, 0x22, 0x19, 0xc7, 0x77, 0x2f
+	},
+	[FBI_BL32_DATA] = {
+		0x34, 0xa1, 0x48, 0xb8, 0xbc, 0x90, 0xe6, 0x11,
+                0x8f, 0xef, 0xa4, 0xba, 0xdb, 0x19, 0xde, 0x03
+	},
+	[FBI_BL33_DATA] = {
+		0x8e, 0x59, 0xd6 ,0x5d, 0x5e, 0x8b, 0xe6, 0x11,
+                0xbc, 0xb5, 0xf0, 0xde, 0xf1, 0x83, 0x72, 0x96,
+	},
+	[FBI_EMPTY] = {},
```

Comment:
Are those only v3 related or are you aware of v2 format that can use them ?

Reply:
AFAIK only v3 related, and no idea how this data is used, probably used when enabling secure boot somehow